diff --git a/node_modules/@wojtekmaj/react-daterange-picker/dist/DateRangePicker.js b/node_modules/@wojtekmaj/react-daterange-picker/dist/DateRangePicker.js
index 545369a..0a661c3 100644
--- a/node_modules/@wojtekmaj/react-daterange-picker/dist/DateRangePicker.js
+++ b/node_modules/@wojtekmaj/react-daterange-picker/dist/DateRangePicker.js
@@ -138,7 +138,7 @@ var DateRangePicker = /*#__PURE__*/function (_PureComponent) {
       var _this$props = _this.props,
           disabled = _this$props.disabled,
           onFocus = _this$props.onFocus,
-          openCalendarOnFocus = _this$props.openCalendarOnFocus;
+          hideCalenderOnInputFocus = _this$props.hideCalenderOnInputFocus;
 
       if (onFocus) {
         onFocus(event);
@@ -147,11 +147,14 @@ var DateRangePicker = /*#__PURE__*/function (_PureComponent) {
 
       if (disabled) {
         return;
-      }
+      } // Allow user to use input without opening the calendar
+
 
-      if (openCalendarOnFocus) {
-        _this.openCalendar();
+      if (hideCalenderOnInputFocus) {
+        return;
       }
+
+      _this.openCalendar();
     });
 
     _defineProperty(_assertThisInitialized(_this), "openCalendar", function () {
@@ -440,7 +443,6 @@ DateRangePicker.defaultProps = {
   closeCalendar: true,
   isOpen: null,
   name: 'daterange',
-  openCalendarOnFocus: true,
   rangeDivider: '–'
 };
 
@@ -459,6 +461,7 @@ DateRangePicker.propTypes = {
   dayPlaceholder: _propTypes["default"].string,
   disableCalendar: _propTypes["default"].bool,
   disabled: _propTypes["default"].bool,
+  hideCalenderOnInputFocus: _propTypes["default"].bool,
   format: _propTypes["default"].string,
   isOpen: _propTypes["default"].bool,
   locale: _propTypes["default"].string,
@@ -473,7 +476,6 @@ DateRangePicker.propTypes = {
   onCalendarOpen: _propTypes["default"].func,
   onChange: _propTypes["default"].func,
   onFocus: _propTypes["default"].func,
-  openCalendarOnFocus: _propTypes["default"].bool,
   rangeDivider: _propTypes["default"].node,
   required: _propTypes["default"].bool,
   returnValue: _propTypes["default"].oneOf(['start', 'end', 'range']),
diff --git a/node_modules/@wojtekmaj/react-daterange-picker/src/DateRangePicker.jsx b/node_modules/@wojtekmaj/react-daterange-picker/src/DateRangePicker.jsx
index 988b058..8649fa7 100644
--- a/node_modules/@wojtekmaj/react-daterange-picker/src/DateRangePicker.jsx
+++ b/node_modules/@wojtekmaj/react-daterange-picker/src/DateRangePicker.jsx
@@ -82,7 +82,7 @@ export default class DateRangePicker extends PureComponent {
   }
 
   onFocus = (event) => {
-    const { disabled, onFocus, openCalendarOnFocus } = this.props;
+    const { disabled, onFocus, hideCalenderOnInputFocus } = this.props;
 
     if (onFocus) {
       onFocus(event);
@@ -93,9 +93,12 @@ export default class DateRangePicker extends PureComponent {
       return;
     }
 
-    if (openCalendarOnFocus) {
-      this.openCalendar();
+    // Allow user to use input without opening the calendar
+    if (hideCalenderOnInputFocus) {
+      return;
     }
+
+    this.openCalendar();
   }
 
   openCalendar = () => {
@@ -334,7 +337,6 @@ DateRangePicker.defaultProps = {
   closeCalendar: true,
   isOpen: null,
   name: 'daterange',
-  openCalendarOnFocus: true,
   rangeDivider: '–',
 };
 
@@ -362,6 +364,7 @@ DateRangePicker.propTypes = {
   dayPlaceholder: PropTypes.string,
   disableCalendar: PropTypes.bool,
   disabled: PropTypes.bool,
+  hideCalenderOnInputFocus: PropTypes.bool,
   format: PropTypes.string,
   isOpen: PropTypes.bool,
   locale: PropTypes.string,
@@ -376,7 +379,6 @@ DateRangePicker.propTypes = {
   onCalendarOpen: PropTypes.func,
   onChange: PropTypes.func,
   onFocus: PropTypes.func,
-  openCalendarOnFocus: PropTypes.bool,
   rangeDivider: PropTypes.node,
   required: PropTypes.bool,
   returnValue: PropTypes.oneOf(['start', 'end', 'range']),
